maximum_subarray/Cpp/max_subarray.cpp

```diff
@@ -0,0 +1,28 @@
+#include<bits/stdc++.h>
+
+using namespace std;
+
+int main(){
+	// Reading size of array and array.
+	cin >> n;
+	
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.cpp`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmplt3zxdi2/maximum_subarray/Cpp/max_subarray.cpp
+++ b/tmp/tmplt3zxdi2/maximum_subarray/Cpp/max_subarray.cpp
@@ -5,7 +5,7 @@
 int main(){
 	// Reading size of array and array.
 	cin >> n;
-	
+
 	for(int i=1;i<=n;i++)
 		cin >> arr[i];
 
```